client/ayon_core/hooks/pre_remove_launcher_paths.py

```diff
@@ -0,0 +1,36 @@
+""""Pre launch hook to remove launcher paths from the system."""
+import os
+from ayon_applications import PreLaunchHook, LaunchTypes
+
+
+class PreRemoveLauncherPaths(PreLaunchHook):
+    """Remove launcher paths from the system.
+
+    This hook is used to remove launcher paths from the system before launching
+    an application. It is used to ensure that the application is launched with
+    the correct environment variables. Especially for Windows, where
+    paths in `PATH` are used to load DLLs. This is important to avoid
+    conflicts with other applications that may have the same DLLs in their
+    paths.
+    """
+
+    order = 1
+
+    platforms = {"linux", "windows", "darwin"}
+    launch_types = {LaunchTypes.local}
```

Comment:
Shouldn't this happen all the time?
```suggestion
```

Reply:
is it set on farm? I thought not

Reply:
Well, `extractenvironments` is using AYON launcher, so yes...